error/src/out_point.rs

```diff
@@ -0,0 +1,79 @@
+use failure::Fail;
+use numext_fixed_hash::H256;
+use std::fmt::{Debug, Formatter, Result as FmtResult};
+
+// NOTE: To avoid circle dependency, we re-define another `ckb_core::OutPoint`,
```

Comment:
I thought that in the discussion, `ckb-error` does not define any context. Each module will add the context instead.